### DISCUSSION TITLE
probleme de license

### DISCUSSION THREAD

**Discussion by Faucheuze** [aclap-dev/video-downloadhelper#1489] [License & Purchase requests]
### What's your issue?

Bonsoir, 
je viens de m'acheter un nouvelle ordinateur, et quand j'essaie de mettre ma clé de license dans l'application "downloadhelper" cela m'indique que la clé est non enregistré. Mon nom est Farid Meziani.

Merci pour votre aide

# VOTRE IMAGE A ETE EFFACEE, ELLE CONTENAIT VOTRE NUMERO DE LICENSE

### The 10 first digits (only 10) of your license number or email address

4509fe9d0b fmezprod@gmail.com

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by jcfrog** [comment]
Bonjour, avez vous cliqué sur le lien en dessous de la license "Check license key" ? https://github.com/aclap-dev/video-downloadhelper/wiki/V9-interface#validate-your-license-key

**Comment by Faucheuze** [comment]
Bonjour,

c'est bon, merci pour ce lien.

Bonne journée.

Cordialement,

Farid
